### PR TITLE
std

### PR DESCRIPTION
**Pull request by apprehensions**
* cleanup ifs
* remove comments
* deprecate Errc()
* config: move dirs to dirs.go
* config: defConfig() return only
* config: seperate retrieval of editor
* util: logging for vinegar
* util: move dirs related funcs to dirs.go
* util: move config related to config.go
* util: only unzip target
* dirs: cwd dir for Exec()
* dirs: set permission if not set on existing dir
* main: handle SIGINT
* main: version & rbxfpsunlocker from argument
* mk: version
* roblox: check renderer config if valid
* thirdparty: set var val
* thirdparty: move rco to roblox.go
* thirdparty: split up dxvk extraction process
* thirdparty: update wineprefix on dxvk dll removal
* thirdparty: split up rbxfpsunlocker settings applier,
* thirdparty: always use current rbxfpsunlocker config file if existing
* wine: init wineprefix func, wineprefix version setter

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Build
+on:
```
Comment (Thelolguy1) [inline .github/workflows/build.yml:2]: This action is unnecessary, we already have the Makefile CI action.

https://github.com/vinegarhq/vinegar/actions/workflows/makefile.yml

Comment (apprehensions) [reply .github/workflows/build.yml:2]: ............. it replaces it.

i hadn't did `git add` properly.

Comment (apprehensions) [reply .github/workflows/build.yml:2]: it did replace it dude lol

**Comment by Thelolguy1** [comment]
pending test

**Review by Thelolguy1 — APPROVED** [review]

**Pull request merged**